microphone/drv_audio_pdm.h

```diff
@@ -8,7 +8,7 @@
 
 #define PDM_BUF_NUM 	2
 #define PDM_BUF_SIZE 	2048
-#define DECIMATION		32
+#define DECIMATION		16
```

Comment:
Please undo this change, this is also fixed in Jose's PR, so better to avoid conflicts and let him handle it.